cherryontech/jupiter-jumpers
# Calendar

## Description

1) Updates the calendar style to include the icons: when a user logs a mood, the icon shows up in that selected tile
2) The selected tile's date is displayed
3) The mood log modal only pops up if the user first selects a tile and then clicks "Add Log"

## Related Issue

TBD

## Related Story Card


## Type of Changes

Feature update, styling update

## Acceptance Criteria

TBD

## Update Screenshots

_Provide screenshots of the view before and after the changes._

### Before

### After
![Screenshot 2024-05-12 at 8 12 39 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/c6719af7-5659-4024-bbe8-40c18001c3b0)

## Testing Instructions

1) Log in
2) Select today's tile
3) Ensure you see "Today" displayed next to the "Add Log" button
4) Attempt to select a day other than today's date, and ensure you see that updated date next to the "Add Log" button
5) Attempt to log a mood for a day. Ensure the icon updates correctly. 
6) Verify that future date tiles are not clickable
7) Attempt to re-record a mood for a day on which a mood was already logged; ensure the update works.
8) Log out, and log back in, and ensure your recorded moods are still there.

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
